extensions/packageManager/src/node/index.ts

```diff
@@ -57,47 +57,48 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
       }
     },
     getFeeds: async function (req, res) {
-      // read the list of sources from the config file.
-      let packageSources: IPackageSource[] = composer.store.read('feeds') as IPackageSource[];
-
-      // if no sources are in the config file, set the default list to our 1st party feed.
-      if (!packageSources) {
-        packageSources = [
-          // TODO: Re-enable the NPM feed when we have a JS runtime
-          // {
-          //   key: 'npm',
-          //   text: 'npm',
-          //   url: 'https://registry.npmjs.org/-/v1/search?text=keywords:bf-component&size=100&from=0',
-          //   searchUrl: 'https://registry.npmjs.org/-/v1/search?text={{keyword}}+keywords:bf-component&size=100&from=0',
-          //   readonly: true,
-          // },
-          {
-            key: 'nuget',
-            text: 'nuget',
-            url: 'https://api.nuget.org/v3/index.json',
-            readonly: true,
-            defaultQuery: {
-              prerelease: true,
-              semVerLevel: '2.0.0',
-              query: 'microsoft.bot.components+tags:bf-component',
-            },
-            type: PackageSourceType.NuGet,
+      // Read the list of sources from the config file.
+      let storedSources: IPackageSource[] = composer.store.read('feeds') as IPackageSource[];
+
+      // Default sources
+      let packageSources: IPackageSource[] = [
+        {
+          key: 'nuget',
+          text: 'nuget',
+          url: 'https://api.nuget.org/v3/index.json',
+          readonly: true,
+          defaultQuery: {
+            prerelease: true,
+            semVerLevel: '2.0.0',
+            query: 'microsoft.bot.components+tags:msbot-component',
           },
-          {
-            key: 'nuget-community',
-            text: 'community packages',
-            url: 'https://api.nuget.org/v3/index.json',
-            defaultQuery: {
-              prerelease: true,
-              semVerLevel: '2.0.0',
-              query: 'tags:bf-component',
-            },
-            type: PackageSourceType.NuGet,
+          type: PackageSourceType.NuGet,
+        },
+        {
+          key: 'nuget-community',
+          text: 'community packages',
```

Comment:
is this string rendered in the ui?

Reply:
@cwhitten I think it is the text that shows up in the drop down where you select feeds and also in the list when you manage your feeds.

Reply:
We need to wrap this in our translation api `formatMessage`. Here is an example in the client: https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/packages/adaptive-flow/src/adaptive-flow-editor/renderers/NodeMenu.tsx#L33

We typically try and encapsulate view data from the server code, of which this is defined. I don't see an issue pulling in `format-message` into scope here, though.